mesonbuild/dependencies/base.py

```diff
@@ -42,6 +42,8 @@ class DependencyMethods(Enum):
     AUTO = 'auto'
     PKGCONFIG = 'pkg-config'
     CMAKE = 'cmake'
+    # The dependency is provided by the standard library and does not need to be linked
+    BUILTIN = 'builtin'
```

Comment:
Perhaps this distinction needs to be memorialized in the discussion of `method:` in Dependencies.md?

Reply:
Fixed, I think.